### PR TITLE
Sync Dev

### PR DESCRIPTION
**Pull request by PythonSmall-Q**
- [x] 我已认真阅读贡献指南 (contributing guidelines) 和社区公约 (code of conduct)，并遵循了如何参与页及格式手册页的相应规范。

By making a contribution to this project, I certify that:

The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

The contribution was provided directly to me by some other person who certified 1., 2. or 3. and I have not modified it.

I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by PythonSmall-Q** [comment]
sitemap还没修好，在搞pr的事情

先merge吧

File: .github/workflows/sitemap-automation.yml [hunk 1]
```diff
@@ -18,12 +18,28 @@ jobs:
...
+      run: |
+        echo "sitemap-path = ${{ steps.sitemap.outputs.sitemap-path }}"
+        echo "url-count = ${{ steps.sitemap.outputs.url-count }}"
+        echo "excluded-count = ${{ steps.sitemap.outputs.excluded-count }}"
+    - name: Check outputs
+      if: ${{ steps.cpr.outputs.pull-request-number }}
       run: |
-        git config --global user.name "hendragon-bot[bot]"
-        git config --global user.email "hendragon-bot[bot]@users.github.com"
-        git add sitemap.xml
```
Comment (boomzero) [inline .github/workflows/sitemap-automation.yml:27]: emm

Comment (boomzero) [reply .github/workflows/sitemap-automation.yml:27]: 你甚至没改script

Comment (boomzero) [reply .github/workflows/sitemap-automation.yml:27]: ...

Comment (boomzero) [reply .github/workflows/sitemap-automation.yml:27]: WAIT, BE PATIENT

**Review by boomzero — CHANGES_REQUESTED** [review]

**Comment by PythonSmall-Q** [comment]
image
现在是这个样子的

**Comment by PythonSmall-Q** [comment]
我说了还没搞定，现在会莫名其妙不能创建pr

**Pull request closed without merging**